### ISSUE TITLE
The space is left behind when the hide button slides down.

### ISSUE THREAD

**Issue by aashimgarg** [fluid-project/fluidproject.org#23]
## Describe the bug
When we resize the window by decreasing its windows width to its minimum the hide button slides down leaving the area as it was before
### To reproduce
1. Go to 'https://fluidproject.org/'
2. Resize the window width to minimum.
3. See error

### Expected behavior
The hide button should go down with no space leaving behind.

## Screenshots
![IMG_20200226_230933 (1)](https://user-images.githubusercontent.com/55639487/75374344-212abb80-58f2-11ea-94b8-3a9659755e69.jpg)

### Desktop
 - OS: [e.g. ubuntu 18.04]
 - Browser [e.g. chrome]

**Comment by sachin10101998** [comment]
You can also use the 'Toggle Device toolbar' provided with Chrome DevTools to simulate mobile devices. You should check [this out](https://developers.google.com/web/tools/chrome-devtools/device-mode).

**Comment by jobara** [comment]
When the site is updated with the latest version of Infusion, and UI Options (the tool you are describing now), it will switch to a mobile/small screen layout. Please see the [UI Options demo](https://build.fluidproject.org/infusion/demos/uiOptions/) for an example. In short, this issue should be addressed by updating to recent dev release of Infusion, or to a new stable version if Infusion 3.0 has been released by the time this is worked on.

**Comment by sachin10101998** [comment]
@jobara , Is [UI Options demo](https://build.fluidproject.org/infusion/demos/uiOptions/) using the latest version of infusion?

**Comment by jobara** [comment]
@sachin10101998 it should be built from the latest code from the master branch.

**Comment by sachin10101998** [comment]
@jobara, @jhung, I had a doubt regarding the span of values that different UI options are providing for line spacing. In the [UI Options Demo ](https://build.fluidproject.org/infusion/demos/uiOptions/)and [floe project website](https://floeproject.org/), the line spacing values from 0.7 to 1 cause the list of elements in the table of contents to overlap ( issue described in fluid-project/floeproject.org#110). However, this issue has been solved by placing a cap on line spacing values in [the feedback demo](https://metadata.floeproject.org/demos/feedback/index.html) where it was solved by keeping the values of line spacing between 1 and 2. Does that mean that fluid-project/floeproject.org#110 can be solved by putting a cap on line spacing values so that it's value doesn't go below 1?

**Comment by aashimgarg** [comment]
@sachin10101998 this issue does not relate to any sort of line spacing.

**Comment by jobara** [comment]
@sachin10101998 did you mean to comment on a different issue?

**Comment by sachin10101998** [comment]
Yeah, I tagged [the issue I was referring to](https://github.com/fluid-project/floeproject.org/issues/110). I raised the topic here since you mentioned UI Options Demo, which has a similar problem as that issue I was referring to. 
I'd make a copy of my comment above under the appropriate issue too.

**Comment by aashimgarg** [comment]
> When the site is updated with the latest version of Infusion, and UI Options (the tool you are describing now), it will switch to a mobile/small screen layout. Please see the [UI Options demo](https://build.fluidproject.org/infusion/demos/uiOptions/) for an example. In short, this issue should be addressed by updating to recent dev release of Infusion, or to a new stable version if Infusion 3.0 has been released by the time this is worked on.

@jobara should i close this issue or should i send a PR for this?

**Comment by jobara** [comment]
Maybe you can do some preliminary investigation into how much work it would be to update to the latest dev release of Infusion from NPM.

**Comment by sachin10101998** [comment]
@jobara, isn't 2.0 latest dev release version of Infusion?

**Comment by jobara** [comment]
@sachin10101998 2.0 is the latest stable release. There have been many 3.0 dev releases. If you go to https://www.npmjs.com/package/infusion and go to the "versions" tab you can see a list. You can also see this by running `npm show infusion versions` from the command line. The latest dev release made from master is https://www.npmjs.com/package/infusion/v/3.0.0-dev.20200213T130945Z.c3c22dc4c

**Comment by aashimgarg** [comment]
Ok @jobara .
